docs/topics/testing/tools.txt

```diff
@@ -2136,6 +2136,39 @@ and contents::
             # Verify that the subject of the first message is correct.
             self.assertEqual(mail.outbox[0].subject, "Subject here")
 
+In reality, most emails being created will be HTML emails, meaning they not
+only contain a plain-text body but an HTML part as well::
```

Comment:
This is a little bit opinionated, maybe let's update to:
```suggestion
Here is another example testing an email with
:ref:`multiple content types <topic-alternative-content-types>`::
```
For this to work we need to create the `topic-alternative-content-types` link. This can be done by updating `docs/topics/email.txt`
```diff
--- a/docs/topics/email.txt
+++ b/docs/topics/email.txt
@@ -379,6 +379,8 @@ The class has the following methods:
   For MIME types starting with :mimetype:`text/`, binary data is handled as in
   ``attach()``.

+.. _topic-alternative-content-types:
+
 Sending alternative content types
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
@GitRon this bit got missed 👍 

> For this to work we need to create the `topic-alternative-content-types` link. This can be done by updating `docs/topics/email.txt`
> 
> ```diff
> --- a/docs/topics/email.txt
> +++ b/docs/topics/email.txt
> @@ -379,6 +379,8 @@ The class has the following methods:
>    For MIME types starting with :mimetype:`text/`, binary data is handled as in
>    ``attach()``.
> 
> +.. _topic-alternative-content-types:
> +
>  Sending alternative content types
>  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> ```

Reply:
Oh damn! Thanks! Let's see if it works now.